logger/common_windows.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// +build windows
+
+package logger
+
+import (
+	"errors"
+)
+
+// Not implemented in Windows
+func setUID(id int) error {
+	return errors.New("Not supported in Windows")
```

Comment:
It would be clear to revise the message clearer as "UID not supported in Windows"?

Reply:
I'll revise the comment.